articles.tex

```diff
@@ -545,9 +545,14 @@
 
 \asection{Appeals}
 Decisions made by the Executive Board may be appealed and overturned.
+\asubsection{Executive Board Appeal}
 To initiate an appeal, a member must have the support of three voting members of the Executive Board, or a petition with the signatures of one-third of Active Members.
 After the appeal is presented, a Simple Majority vote is held by the Executive Board whether or not to overturn and reevaluate the decision.
 If the vote passes, the Executive Board may discuss and make a new ruling by another Simple Majority vote.
+\asubsection{House Appeal} 
+To initiate an appeal to House of an Executive Board decision, a petition with the signatures of two-thirds of Active Members must be brought to a House Meeting. 
+After the appeal is presented and discussed, a Two-Thirds vote is held by the attending members. 
```

Comment:
Agreeing with @gambogi please add some kind of quorum here. Perhaps `A quorum of two-thirds of the Total Number of Possible Votes`